### ISSUE TITLE
Arastta servers down, or access blocked - Installer doesn't work

### ISSUE THREAD

**Issue by syntaxerror** [arastta/arastta#761]
Related to issue #740, who obviously was only partial fixed ...

When the arastta.io server is down, or not available from the server Arastta is being installed on, installer fails to load because of the getLanguages function:
https://github.com/arastta/arastta/blob/1.7-dev/install/model/language.php#L13

_For those having trouble installing, the temporary solution is to comment line 17 to 27 in the file mentioned above._

**Comment by denisdulici** [comment]
Issue fixed.

**Comment by syntaxerror** [comment]
In what way? The code is not changed for this, so it would still be a problem next time the server is gone or blocked. Or?

Please reopen untill the code is fixed to allow installation, even if arastta.io is down.

**Comment by denisdulici** [comment]
arastta.io is up and running now. Will release the 1.7 version soon with the current fixed issues.

**Comment by syntaxerror** [comment]
Yes, but the installer is not in that fix Denis, it's only for the admin side - update check and extension installer. This is another one, also mentioned in the original issue, related to installing Arastta itself.

If you test changing to a non working URL here: https://github.com/arastta/arastta/blob/1.7-dev/install/model/language.php#L18

And then tries to install Arastta, it will most likely return just a blanck page, or a HTTP ERROR 500 (depending on server config).

So a similar fix should be added to the installer, as the one used for update function and extension installer here https://github.com/arastta/arastta/commit/f12f54ee887274832ce61ac124a07694eb105f38

**Comment by denisdulici** [comment]
Hmm... You're right, opening this back then
